src/FixedDataTable.js

```diff
@@ -440,11 +439,6 @@ class FixedDataTable extends React.Component {
      */
     onColumnReorderEndCallback: PropTypes.func,
 
-    /**
-     * Whether a column is currently being resized.
-     */
-    isColumnResizing: PropTypes.bool,
```

Comment:
Wouldn't this prop still be used by the older API?
Also, can you mark the other props as deprecated? Including it in the comment should be fine, since the comment will be reflected into our API docs.

Reply:
Yes, these props will still be used in the older API. So I should just add these back and add a label `deprecated`?

Reply:
No, it won't be used.